meetings/2022-06-30.md

```diff
@@ -39,38 +42,99 @@
 
 ### WIP Implementation Feedback
 * SC: We'll allocate some time for implementation feedback going forward. Links to servers/applications welcome.
+* CB: Heads up that Apple announced iOS16 support for Web Push notifications. So it'll work across all devices. Safari didn't support but on MacOS they added 1-2 years ago but... people wanted it so now they know. The community is developers was very focused and Apple gave in.
+* SC: More adoption is good sign.
+* RG: What are you mainly developing when it comes to push notifications.
+* SC: I'm not currently developing push notifications but possible because dokieli can be used as a browser extension so that might help out to indicate when there is an annotation available on a document.
+* RG: Is there a server implementation that's going hand in hand with the notifications protocol we are developing here.
+* SC: So there are private and public implementations - some are experimental and some are waiting for a more stable version of the spec. I think this will happen in the next months when the Notifications Protocol will stabilise as much as it could any way, as well as the Solid Protocol. It's ongoing work.
+* CB: I'd like add re implementation, I have the demo implementation for Web Push. This works in a demo-way but there is still the issue how we model the subscription resource (Subscription resource as a container #[36](https://github.com/solid/notifications/issues/36) ). Resolving this issue will help to adapt my implementation.
+* SC: You should insist on feedback. If I miss in the agenda, propose as topic or use Gitter chat.
+* RG: will review after meeting.
 
 
 ### Discovery of Notification Subscription Metadata
 URL: https://github.com/solid/notifications/issues/58
 
 * SC: There is now `solid:storageDescription` in Solid Protocol ED ( https://github.com/solid/specification/pull/416 , https://solidproject.org/ED/protocol#server-storage-description )
-
-
+* SC: This adresses some of the considerations wrt #58, how much we rely on the Solid Protocol and how much the Notifications Protocol covers.
+* SC: There might be related comments in #76.
+* SC: Discovering the features that are available gives a good idea of what is missing. It doesn't matter where that place for discovery is, but now we have a reliable place where this information lives with the `solid:storageDescription`.
+* SC: If the Notifications Protocol relies on that feature, then it can say what the channel/subscription information should be available there.
```

Comment:
```suggestion
* SC: If the Notifications Protocol relies on that feature, then it can say what channel/subscription information should be available there.
```